dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSTLTests.java

```diff
@@ -183,4 +190,41 @@ public void testFormatTags() throws Exception {
 
 
     }
+
+    /**
+     * Flushes during an import. See below for more details. 
+     *  
+     * https://github.com/eclipse-ee4j/jstl-api/issues/153
+     *
+     * @throws Exception if something goes horribly wrong
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    public void testImportFlushing() throws Exception {
+        WebConversation wc = new WebConversation();
+
+        // Imported file should stay together even if a flush is called 
+        // Begin
+        //
+        // End
+        // String expectedString = String.join(newLine,
+        //                "Begin",
+        //                "",
+        //                "End");
+
+        String url = JSPUtils.createHttpUrlString(server, IMPORT_APP_NAME, "index.jsp");
+        LOG.info("url: " + url);
+
+        WebRequest request = new GetMethodWebRequest(url);
+        WebResponse response = wc.getResponse(request);
+
+        String text =  response.getText();
+        LOG.info("Servlet response : " + response.getText());
+
+        int beginIndex = text.indexOf("Begin"); // 1006
+        int endIndex = text.indexOf("End"); // 1014
+
+        assertTrue("Something went wrong with the flushing within the import tag", 8 == (endIndex - beginIndex));
```

Comment:
Checking for multi line string proved difficult, so I looked at the indices instead.